src/components/PersonLink.tsx

```diff
@@ -0,0 +1,69 @@
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person,
+  motherNameLink: string | undefined,
+  fatherNameLink: string | undefined,
+};
+
+export const PersonLink: React.FC<Props> = ({
+  person,
+  motherNameLink,
+  fatherNameLink,
+}) => {
+  const { slug } = useParams();
+
+  return (
+    <tr
+      key={person.slug}
```

Comment:
```suggestion
```
Key should be added only in parent component, no need to duplicate it here

Reply:
Thank you!